docs/codeathon.md

```diff
@@ -0,0 +1,25 @@
+# FAIR Codeathon 2021 Helpful Information
+
+![codeathon](https://images.ctfassets.net/6bya4tyw8399/2qgsOmFnm7wYIfRrPrqbgx/ae3255858aa12bfcebb52e95c7cacffe/codeathon-graphic.png)
+
+For participants of the [FAIR Codeathon](https://sparc.science/help/2021-sparc-fair-codeathon) July 12-27 2021, we have compiled the parts of the documentation that are likely to be most useful to you. As a very brief introduction, 
+
+**o<sup>2</sup>S<sup>2</sup>PARC** is a cloud-based computational platform for running code that is contained in various modules, called [services](docs/platform_introduction/core_elements/services.md). These services can be chained together which allows for file-based information transfer (see [Connecting Services](docs/study_setup/connecting_services.md)) from one service to another. A project on **o<sup>2</sup>S<sup>2</sup>PARC**, which contains these chained pipelines is called a [study](docs/platform_introduction/core_elements/studies.md).
+
+### Using Services
+Services come in generally two flavors: computational or interactive. Computational services such as a [Python Runner](docs/tutorials/python_runner.md) will allow setting of some parameters (e.g. input files) and will perform a computation with no user interaction required. Interactive services only perform tasks with user interaction, as is the case for services such as the [JupyterLab](docs/study_setup/JupyterLabs.md). JupyterLab services will allow you to write and execute code within the JupyterLab GUI, but as with all interactive services, will not run automatically when a study pipeline is [launched](docs/study_setup/run_simulation.md). Computational services, on the other hand, will automatically read all input parameters at runtime and execute the static code embedded inside them. 
+
+### API 
+Extensive information about the **o<sup>2</sup>S<sup>2</sup>PARC** API is documented [here](https://itisfoundation.github.io/osparc-simcore-python-client/#/) and it was created to allow for running of pipelines through a Python client. Certain issues to keep in mind, however, are that:
```

Comment:
I would highlight the introduction tutorial ... it is a little section of the doc and can be easily missed

Reply:
done